mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -196,7 +199,7 @@ class SetpointRawPlugin : public MavRosPlugin,
 				position.x(), position.y(), position.z(),
 				velocity.x(), velocity.y(), velocity.z(),
 				af.x(), af.y(), af.z(),
-				req->yaw, req->yaw_rate);

```

Comment:
I don't think that adding overloaded function here is good idea - with vector3 or quaternion you know what expect from transforms. With single double - nope (it is roll, pitch or yaw?).